core/src/main/resources/hudson/PluginManager/installed.jelly

```diff
@@ -161,16 +161,22 @@ THE SOFTWARE.
                 </tr>
               </j:forEach>
               <!-- failed ones -->
-              <j:forEach var="p" items="${it.pluginManager.failedPlugins}">
+              <j:forEach var="p" items="${it.failedPlugins}">
                 <tr class="hoverback">
-                  <td class="pane" />
+                  <td class="center pane enable">
+                    <input type="checkbox" disabled="disabled"/>
+                  </td>
                   <td class="pane">
-                    <h4 class="error">Failed : ${p.name}</h4>
-                    <div style="padding-left: 1em">
-                      <pre>${p.exceptionString}</pre>
+                    <div>
+                        ${p.name}
```

Comment:
Shouldn't this be formatted the same as the other plugin names, including the hyperlink to the information about the plugin? i (Untested, me guessing that the formatting should be the same)
```suggestion
                    <div>
                      <a href="${p.url}" target="_blank" class="display-name">
                        ${p.updateInfo.displayName?:p.displayName}
                      </a>
                    </div>
<!--                        ${p.name} -->
```

Reply:
Nice try. Been there too.

https://github.com/jenkinsci/jenkins/blob/07c48b03fb4c651e7314e7adbe3a0ef0fe249586/core/src/main/java/hudson/PluginManager.java#L2119-L2131

Plus tons of callers, some of which have nothing other than an `.hpi` file name and an exception.

Reply:
Thanks for the reply.  Clear evidence that I didn't actually try to make it work.  What you've done is a significant improvement.  Layout that looks different than the rest is still dramatically better than complete absence of the plugin from the list.